### ISSUE TITLE
Create Userflow for Aynsychronous Appt Scheduling Notification

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#96417]

**Comment by loripusey** [comment]
@cgednalske @zach-park It appears that we may not need this ticket; I think we need a team discussion first on implementation options; @cgednalske can you set one up for UX, Product, & BE to discuss?

**Comment by cgednalske** [comment]
Sure thing! Is that change based on Lee's findings yesterday with how quick the polling occurs? If I'm reading that currently and it's that quick, then definitely sounds like we could just use a standard page loading indicator as we receive the info.

**Comment by loripusey** [comment]
I think we also have to consider the zero-silent-failures effort.

**Comment by cgednalske** [comment]
Sounds good to discuss. I can't quick tell how quick the process is from Lee's threads. 👀  I'll set something up 👍

**Comment by cgednalske** [comment]
@loripusey I added in some information based on the discussion today

**Comment by cgednalske** [comment]
@Cieramaddox @StevenStraily want to comment your point estimates here?

**Comment by cgednalske** [comment]
@Cieramaddox @StevenStraily reminder to add your point estimates for this when you get a chance.

**Comment by StevenStraily** [comment]
I'm going to estimate 3, and my apologies for missing the notifications on this one.

**Comment by cgednalske** [comment]
Sounds good! We'll ask Lori how she wants to point this on our board, seeing as you and Ciera are splitting it/collaborating.

**Comment by loripusey** [comment]
I will put it at 2 points on our board for Steve's work

**Comment by cgednalske** [comment]
Note: based on the conversations we'll have today (and coming up), this work might be subject to change.

**Comment by cgednalske** [comment]
Yes, we'll will need to send a notification. I think this is still in progress @Cieramaddox @StevenStraily

**Comment by StevenStraily** [comment]
Here are the [user flows](https://www.figma.com/design/DsRXEFiYLCFnY5nBkp9Dc4/CC-Referral-%7C-Appointments-FE?node-id=6133-11630&t=oaWMEzooXYapjY0x-4) for the scenarios we talked about. Here is [the Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1732300978647/b9f97dfb5c92d9633c21cbdefc2f7b30a3e4192e) for the second user flow as well.

The second flow shows that we would send the SMS message regardless and also provides the option for sending a failure message and email if that is something we decide to opt into.
